runtimes/pythonrt/py-sdk/autokitteh/atlassian.py

```diff
@@ -15,7 +14,7 @@
 __TOKEN_URL = "https://auth.atlassian.com/oauth/token"
 
 
-def atlassian_jira_client(connection: str, **kwargs) -> Jira:
+def jira_client(connection: str, **kwargs) -> Jira:
     """Initialize an Atlassian Jira client, based on an AutoKitteh connection.
 
     API reference:
```

Comment:
Missing docstring for the arguments, see the original docstring.

Reply:
I don't understand what you mean. Missing docstring in `jira_client` ?
Do you mean this?

>    Args:
        connection: AutoKitteh connection name.

Reply:
Ah, github diff clipped this. Please document `kwargs`